model_analyzer/triton/model/model_config.py

```diff
@@ -183,22 +183,16 @@ def write_config_to_file(self, model_path, src_model_path,
         model_config_bytes = text_format.MessageToBytes(self._model_config)
         # Create current variant model as symlinks to first variant model
         if first_variant_model_path is not None:
-            src_model_name = Path(src_model_path).name
-            first_variant_model_name = Path(first_variant_model_path).name
-
-            # If the model files have been copied once, do not copy it again
-            if re.search(f'^{src_model_name}_config\d+$',
```

Comment:
@matthewkotila I know you touched code around here recently. Do I need this code that I ripped out? In what case would it kick in and skip?

Reply:
Why did we remove this regex check?
It was checking to make sure we didn't copy files multiple times. It would create a symlink to the first variant for a subset of the output files that don't change between variants.

Reply:
Yes, I've manually run and all tests are passing

Reply:
then we should be good to remove it 👍

Reply:
works for me.